source/clear-linux/tutorials/azure.rst

```diff
@@ -1,125 +1,494 @@
+       mysql            : Manage Azure Database for MySQL servers.
+       network          : Manage Azure Network resources.
+       policy           : Manage resource policies.
+       postgres         : Manage Azure Database for PostgreSQL servers.
+       provider         : Manage resource providers.
+       redis            : Access to a secure, dedicated Redis cache for your Azure applications.
+       reservations     : Manage Azure Reservations.
+       resource         : Manage Azure resources.
+       role             : Manage user roles for access control with Azure Active Directory and service
+                          principals.
+       sf               : Manage and administer Azure Service Fabric clusters.
+       snapshot         : Manage point-in-time copies of managed disks, native blobs, or other
+                          snapshots.
+       sql              : Manage Azure SQL Databases and Data Warehouses.
+       storage          : Manage Azure Cloud Storage resources.
+       tag              : Manage resource tags.
+       vm               : Provision Linux or Windows virtual machines.
+       vmss             : Manage groupings of virtual machines in an Azure Virtual Machine Scale Set
+                          (VMSS).
+       webapp           : Manage web apps.
+
+Log into your Microsoft Azure account
+*************************************
+
+With the :command:`az` command properly installed and functional, login to
+your MS Azure account using the :command:`az login` command. You will be
+prompted to open your browser and enter the displayed URL and enter the code
+XXXXXXXXX to authenticate, where XXXXXXXXX is a random code generated for each
+session.
+
+.. code-block:: console
+
+   $ az login
+
+The output from this command is:
+
+.. code-block:: console
+
+   To sign in, use a web browser to open the page https://aka.ms/devicelogin and enter the code XXXXXXXXX to authenticate.
+
+Following the instructions, the website takes you to a MS Azure device login
+page and asks you to enter the generated code. Enter the code and the
+website changes to a browser screen to enter your existing Microsoft Azure
+credentials. Log in with your Azure account credentials. Once complete, the
+browser screen changes, telling you that you have signed in to the Microsoft
+Cross-platform Command Line Interface application on your device and you can
+close the window. The MS Azure CLI 2.0 interface is now active with your
+Azure account information.
+
+Locate the Clear Linux image
+****************************
+
+You can locate the available clear linux images in the MS Azure marketplace by
+running the following :command:`az` command:
+
+.. code-block:: console
+
+   $ az vm image list --offer clear-linux --all --output table
+
+This command may take some time to finish and the output lists all available
+|CL| images available in the Microsoft Azure marketplace and is shown below:
```

Comment:
Do not add the entire list of available images. That is a nightmare to maintain. I would end the sentence after marketplace